### DISCUSSION TITLE
幫忙推廣一下以前高中學校開的解題專區

### DISCUSSION THREAD

**Discussion by v0ouo** [1101-GS4719-C/Python-Community-GS4719-C#1613] [General]
應該不少人都有聽過Zerojudge吧!那是一個有許多題目的onlinejudge系統，那不知道有沒有人聽過dandanjudge呢?dandanjudge是桃園市私立復旦高級中學所開的，裡面有各式各樣的題目，一開始只有一些人能出題，後來越來越多人出了各式各樣的題目，有很好解的水題，也有給電神解的題目，在大家以及我們學校程設班的努力下，題目編號終於變成b開頭了呢!
![螢幕擷取畫面 2023-10-04 131752](https://github.com/NCU-GS4719-Python/Python-Community/assets/144879969/a7cbc1b0-3b29-4339-9bbb-7641c7f2bad5)

另外有沒有人要來解小弟出的題目，非常水的啦
![螢幕擷取畫面 2023-10-04 132614](https://github.com/NCU-GS4719-Python/Python-Community/assets/144879969/d93907ed-ecea-420c-939d-0818d7fba9a6)

最後，非常不推薦大家就讀桃園市私立復旦高級中學，但非常推薦大家去復旦程式設計班看看

**Comment by eric202501** [comment]
不愧是我們復旦高級中學的電神，不僅會coding還是音遊大神，還是音遊社的骨幹核心，如果去那個網站刷題，也可以像你一樣強嗎
